### PR TITLE
BaseLayerSwitcher Animation Update

### PR DESCRIPTION
**Pull request by danji90**
# How to
The BaseLAyerSwitcher opening/closing animation needs to be updated based on the style in https://map.geo.admin.ch/.
Successfully tested locally in Trafimage-Maps 

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [x] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] IE11 tested.
- [x] Labels applied. if it's a release? a hotfix?
- [x] Tests added.

### CODE REVIEW

File: src/components/BaseLayerSwitcher/BaseLayerSwitcher.js [hunk 1]
```diff
@@ -107,7 +107,7 @@ function BaseLayerSwitcher({
     : baseLayers.map((layer) => layer.get('previewImage'));
 
   const openClass = switcherOpen ? ' rs-open' : '';
-  const closedClass = isClosed ? ' rs-closed' : '';
+  const hiddenStyle = switcherOpen && !isClosed ? 'visible' : 'hidden';
```
Comment (oterral) [inline src/components/BaseLayerSwitcher/BaseLayerSwitcher.js:110]: use rs prefix pls

File: src/components/BaseLayerSwitcher/BaseLayerSwitcher.js [hunk 2]
```diff
@@ -146,41 +148,58 @@ function BaseLayerSwitcher({
...
-          const layerName = layer.getName();
-          const activeClass =
-            layerName === currentLayer.getName() ? ' rs-active' : '';
-          const imageStyle = getImageStyle(
-            layerImages
-              ? layerImages[`${layer.key}`]
-              : layer.get('previewImage'),
-          );
-          return (
+      {
```
Comment (oterral) [inline src/components/BaseLayerSwitcher/BaseLayerSwitcher.js:169]: uselss I guess

File: src/components/BaseLayerSwitcher/BaseLayerSwitcher.js (hunk 1 above)
Comment (danji90) [reply src/components/BaseLayerSwitcher/BaseLayerSwitcher.js:110]: Nope, these are not classes, but strings applied to the style object used in react, so adding a prefix will break it

File: src/components/BaseLayerSwitcher/BaseLayerSwitcher.js (hunk 2 above)
Comment (danji90) [reply src/components/BaseLayerSwitcher/BaseLayerSwitcher.js:169]: completely useless

**Comment by danji90** [comment]
> Looks very smooth .
> 
> The last problem I see is the width:600px size of wrapper. You can't move the map. Any chance to calculate this width automatically? or to reduce it? or to allow the mouse events to go to the map?

Yes you're right, I forgot to add this fix. Will try to find a CSS solution

**Comment by danji90** [comment]
@oterral I removed the pointer events from the parent container so they pass through to the map. Please give it a last review and let me know when to merge

**Comment by gevos** [comment]
> I removed the pointer events from the parent container so they pass through to the map. Please give it a last review and let me know when to merge

Looks good.

**Pull request merged**